quipucords/scanner/openshift/entities.py

```diff
@@ -95,6 +95,7 @@ class OCPNode(OCPBaseEntity):
     operating_system: str = None
     taints: List[dict] = None
     errors: Dict[str, OCPError] = Field(default_factory=dict)
+    unschedulable: bool = None
```

Comment:
```suggestion
    unschedulable: bool = False
```

Following the previous comment you cited, I think `False` is a better default than `None`.

Reply:
I could, but what if the object was initialized without this info. Could we infer it is indeed unschedulable?

Business logic for OCP scanner are concentrated in inspect and api modules.